test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -23,9 +23,58 @@
 using namespace std;
 using namespace yul::test::yul_fuzzer;
 
+const std::string yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
```

Comment:
```suggestion
std::string const yul::test::yul_fuzzer::createHex(std::string const& _hexBytes)
```
Not completely sure though (about which style is correct)

Reply:
@leonardoalt I think your suggestion is correct.

Reply:
Thank you

Reply:
I actually don't think returning the string as a const value has any benefit. It actually might add another copy.